invenio_users_resources/records/hooks.py

```diff
@@ -52,15 +50,18 @@ def post_commit(sender, session):
     # DB operations are allowed here, not even lazy-loading of
     # properties!
     sid = id(session)
-    for user_id in current_db_change_history.updated_users[sid]:
-        reindex_user.delay(user_id)
+    if current_db_change_history.sessions.get(sid):
```

Comment:
here we said that it would be best to spawn only one celery task, that will reindex all affected users/roles, instead of `n` tasks. Was it problematic to change?